todo/src/main/java/com/statsig/todo/TodoApplication.java

```diff
@@ -33,13 +33,11 @@ public static void main(String[] args) throws Exception {
 //		disableCertificateValidation();
 //		disable4();
 //
-//
-//
+//		//System.setProperty("javax.net.ssl.trustStore", "");
```

Comment:
Can we delete all these comments?

Reply:
Yes Daniel, in final PR for this server-side change, I will remove all the unwanted code which I added for the debugging.